src/ORM/Table.php

```diff
@@ -2304,4 +2305,28 @@ public function __debugInfo()
             'connectionName' => $conn ? $conn->configName() : null
         ];
     }
+
+    /**
+     * Validates that a belongsToMany or hasMany assoc has at least one entry.
+     *
+     * This can be used to validate any other array as well.
+     *
+     * @param mixed $value
+     * @return boolean
+     */
+    public function validateCount($value, $expectedCount = 0, $operator = '>')
```

Comment:
I've asked for this as well before:

If we put this into the Validation class, do we want to keep the _ids in there as well because this is specific to assocs. A solution might be to use Hash::path() and to provide the path to the validation method.